test/functional/cmdLineTests/jvmtitests/playlist.xml

```diff
@@ -207,6 +207,36 @@
 			<type>native</type>
 		</types>
 	</test>
+
+
+	<test>
```

Comment:
@smlambert Does this need to be a completely separate test? If it's fast, I see no reason not to include it in the sanity JVMTI test suite, rather than running it separately.

Reply:
I am fine to have it as part of the sanity suite instead of extended, though considering both suites now get run daily and both must pass (we no longer have the notion of non-gold, I think it is also sufficient to keep it in the extended level.  Should we decide to move it to sanity, we can change line 229 below to be `<level>sanity</level>`